docs/explanations/getting-started.md

```diff
@@ -5,34 +5,52 @@ This chapter will briefly introduce a few concepts that should help you learn QI
-Much of this functionality is not unique to microbiome marker gene analysis, but rather general to biological data science, and as result the scope of QIIME 2 is now broader than when we started.
-So, what is QIIME 2?
+Much of this functionality is not unique to microbiome marker gene analysis, but rather general to biological data science, and as result the scope of QIIME 2 became broader than when we started.
 
-What most people think of as "QIIME 2" is what we refer to in our documentation as the *amplicon distribution of QIIME 2*, or simply the *amplicon distribution*.
+What most people think of as "QIIME 2" is what we've referred to in our documentation as the *amplicon distribution of QIIME 2*, or simply the *amplicon distribution*.
+For clarity, going forward, we will refer to this as *QIIME 2*.
 This is the microbiome marker gene analysis toolkit.
-**This documentation site is for the *amplicon distribution* specifically, and we're going to get to that very shortly.**
+**This documentation site is for *QIIME 2* (formerly the *amplicon distribution*) specifically, and we're going to get to that very shortly.**
 
-The *amplicon distribution* is built on what we call the QIIME 2 Framework (or *the framework*).
-The framework is where the general-purpose functionality exists, including data provenance tracking, the plugin manager, and more.
-As an end user, you don't really need to know anything about this, but it's helpful to know that it exists and is different from the amplicon distribution to understand the ecosystem of tools.
-The amplicon distribution, and other tools such as [MOSHPIT](https://moshpit.readthedocs.io/) (formerly referred to as the *metagenome distribution*) and [genome-sampler](https://genome-sampler.readthedocs.io/), are technically *built on top of the QIIME 2 Framework*.
+*QIIME 2* is built on what we called the QIIME 2 Framework (or *the framework*), and are in the process of rebranding as `rachis`.
+From here on, we'll refer to the QIIME 2 Framework as `rachis`.
+`rachis` is where the general-purpose functionality exists, including data provenance tracking, the plugin manager, and more.
+There is no microbiome-specific functionality (or even bioinformatics-specific functionality) in `rachis`.
+As an end user, you don't really need to know anything about this, but it's helpful to know that `rachis` exists and is different from QIIME 2 to understand the ecosystem of tools and how they can build on each other.
+QIIME 2, and other tools such as [MOSHPIT](https://moshpit.readthedocs.io/) (formerly referred to as the *QIIME 2 metagenome distribution*) and [genome-sampler](https://genome-sampler.readthedocs.io/), are technically *built on top of `rachis`*.
```

Comment:
Maybe an example here. 

Distros that are built on top of get all the functionality that makes "QIIME 2" so awesome. cache, provenance tracking, pipeline resumption.